git/git.go

```diff
@@ -71,6 +71,50 @@ func UncommittedChangeCount() (int, error) {
 	return count, nil
 }
 
+type Commit struct {
+	Sha   string
+	Title string
+}
+
+func Commits(baseRef, headRef string) ([]*Commit, error) {
+	logCmd := GitCommand(
+		"log", "--pretty=format:%H,%s",
```

Comment:
Something that I've learned in hub is that we should try to pre-emptively negate the `log.showSignature` setting, otherwise based on the user's settings the `git log` command might try to calculate the validity of signed commits. https://github.com/github/hub/commit/910dfdcb500dd7866939ed65bd5d37d7d42e3356